src/HttpHeader.cc

```diff
@@ -511,28 +509,34 @@ HttpHeader::parse(const char *header_start, size_t hdrLen)
         addEntry(e);
     }
 
-    if (clen.headerWideProblem) {
-        debugs(55, warnOnError, "WARNING: " << clen.headerWideProblem <<
+    if (interpreter.headerWideProblem) {
+        debugs(55, warnOnError, "WARNING: " << interpreter.headerWideProblem <<
                " Content-Length field values in" <<
                Raw("header", header_start, hdrLen));
     }
 
-    if (chunked()) {
+    if (interpreter.prohibited) {
```

Comment:
s/prohibited/prohibitedAndIgnored/ to emphasize that this particular prohibition does not affect message framing.